.github/ISSUE_TEMPLATE/Documentation.md

```diff
@@ -3,7 +3,8 @@ name: Documentation
 about: Report mistakes or request for documentation
 ---
 
-/kind documentation
+/area documentation
+/kind bug
 
```

Comment:
I'm still not sure if  `/kind cleanup` make sense. that is probably for a longer discussion. 
If we decide to do something with cleanup we can do it in a separate PR

Reply:
> I'm still not sure if `/kind cleanup` make sense. that is probably for a longer discussion. If we decide to do something with cleanup we can do it in a separate PR

Then maybe remove the `/kind cleanup` because the label we currently have is deprecated, and keeping it in the template does not make sense.

Reply:
I'm confused :-/
I don't see any `/kind cleanup` in this template for documentation. 

This PR is supposed to clean up a template for documentation issues, nothing else.

If you are talking about a PR template, that is for a separate discussion in another PR.